apis/livepeer/livepeer.go

```diff
@@ -470,6 +477,12 @@ func (lapi *API) GetWebhooksForEvent(userId string, streamId string, event strin
 	return lapi.GetWebhooks(u)
 }
 
+// Update webhook status & response
+func (lapi *API) UpdateWebhookStatus(id string, status map[string]interface{}) bool {
+	go lapi.UpdateWebhook(id, status)
+	return true
```

Comment:
This return value seems unnecessary. Actually the whole function. In go patterns you wouldn't need a function `g()` to do `g() { go f() }`, you just make `go f()` (and it's also clear on call-site that it is async). 

IOW The callers of `UpdateWebhookStatus` can call `go UpdateWebhook()`.

Reply:
Side note, if you prefer to keep the 2 functions, might prefer to rename this helper to have a clearer name, since it's not really describing what it does in relation to the `UpdateWebhook` func. It could be called `UpdateWebhookAsync` or something like that, WDYT?

Reply:
Yeah for now I prefer to keep the two functions, everything works fine and I want to ship it in the separate docker to debug bonfire, will get to this later down the road! Renamed to `async` and removed the boolean return btw